wrappers/python/nntile/nntile_core/tensor.pyi

```diff
@@ -0,0 +1,55 @@
+from collections.abc import Buffer
+from typing import Sequence
+
+from .tile import Tile, TileTraits
+
+class TensorTraits:
+    """Integer arithmetics for tensors."""
+
+    def __init__(self, shape: Sequence[int], basetile_shape: Sequence[int]):
+        ...
+
+    def __repr__(self) -> str: ...
+
+    @property
+    def basetile_shape(self) -> tuple[int, ...]: ...
+
+    @property
+    def grid(self) -> TileTraits: ...
+
+    def get_tile_shape(self) -> tuple[int, ...]: ...
+    def get_grid_shape(self) -> tuple[int, ...]: ...
+
+
+class Tensor(TensorTraits):
+    """Many-dimensional tensor, presented by a set of subtensors (tiles)."""
+
+    def __init__(self, traits: TensorTraits, distribution: Sequence[int],
+                 last_tag: int): ...
+
+    @property
+    def next_tag(self) -> int: ...
+
+    def from_array(self, array: Buffer) -> None: ...
+    def to_array(self) -> Buffer: ...
```

Comment:
This `to_array` function returns value, while other `to_array` functions do not return anything. This is why I was asking to convert other `to_array` functions.